PythonicPenguins/bangazon_api
# move gitignore to the root

## Status
READY

## Migrations
NO

## Description
Move `.gitignore` to the root directory.

## Related PRs
N/A

## Resolves Issue Number
N/A

## Deploy Notes
Checkout to master and run `git pull`

## Steps to Test or Reproduce
After pulling from master branch, run `git status` and make sure that the migrations folder is not showing up as pending to add.

## Impacted Areas in Application
N/A
